### PR TITLE
📝[UPDATE] Discord Bot List & Tatsumaki presence

### PR DESCRIPTION
**Pull request by eren885**
unknown
unknown
unknown

### CODE REVIEW

File: websites/D/Discord Bot List/dist/metadata.json [hunk 1]
```diff
@@ -9,14 +9,14 @@
     "tr": "Discord Bot List bir bot listesidir."
   },
   "url": "discordbotlist.com",
-  "version": "1.2.3",
-  "logo": "https://erens.glitch.me/96So.png",
-  "thumbnail": "https://erens.glitch.me/MxWr.png",
+  "version": "1.2.4",
+  "logo": "https://i.imgur.com/RGAILaY.png",
```
Comment (frozaum) [inline websites/D/Discord Bot List/dist/metadata.json:13]: Hello, can you use a higher-quality version of the logo, thanks.

**Comment by doomlerd** [comment]
Why are you deleting Tatsumaki?

**Comment by eren885** [comment]
> Why are you deleting Tatsumaki?

wait what i didn't mean to do thattt ohhhhh *sighs*

**Comment by eren885** [comment]
i am really stupid, sorry guys.

**Pull request merged**